### DISCUSSION TITLE
White background on dark mode

### DISCUSSION THREAD

**Discussion by LITUATUI** [razonyang/hugo-theme-bootstrap#663] [General]
Now on dark mode the background stays white. This is a bug right?

![dark-mode](https://user-images.githubusercontent.com/49299328/185080397-c5ed193a-7d78-48c7-9ff7-252beef609f4.jpg)

**Comment by razonyang** [comment]
This issue should be fixed by abca9c00511d94c4695c190b5e8724ce8b105fc8 committed yesterday, since the bootstrap dark mode latest commit uses `data-bs-theme` instead of `data-theme`. This can be solved by updating the theme to latest commit.

**Comment by LITUATUI** [comment]
I did update with the "hugo mod get -u" command.

![pushevs-hugo](https://user-images.githubusercontent.com/49299328/185082257-91df7185-efc8-49b5-bc41-3bdb66c16592.jpg)

Also, the sidebar now seems to be on the left side by default. Previously it was on the right. How can I change it back?

Thanks!

**Reply by razonyang** [reply]
Could you please post the `go.mod`? I use `-u` before, but it won't pull the latest commit~ I use `hugo mod get razonyang/hugo-theme-bootstrap@master` for pulling latest commit.

**Comment by LITUATUI** [comment]
You're correct, thanks.

Using "hugo mod get github.com/razonyang/hugo-theme-bootstrap@master" fixed the white background.

How about the sidebar? I can't find on the documentation how to define its side (left or right).

**Reply by razonyang** [reply]
hmm That's weird, the sidebar placed left only on small device, do you have a live site or repo so that I can reproduce the issue?

**Comment by LITUATUI** [comment]
Sorry, my mistake. I was using an old partial. All good now!
